src/main/java/com/tecnalia/datausage/api/VersionController.java

```diff
@@ -11,7 +11,7 @@
 @RequestMapping({ "/about" })
 public class VersionController {
 
-	@Autowired
+
```

Comment:
Any particular reason for deleting annotation?

Since the spring-boot-maven-plugin is configured properly.

Reply:
Field buildProperties in com.tecnalia.datausage.api.VersionController required a bean of type 'org.springframework.boot.info.BuildProperties' that could not be found.

The injection point has the following annotations:
	- @org.springframework.beans.factory.annotation.Autowired(required=true)
	
This is the error when trying to use this property.